Scenes/Levels/TemplateLevel/TemplateLevel.gd

```diff
@@ -1,28 +1,40 @@
 extends Node2D
 
 onready var score_label = $ScoreLabel
+onready var tower_options = $HUD/Debug/TowerOptionButton
+onready var game_manager = get_node("/root/GameManager")
```

Comment:
Assuming it is enabled in the autoload settings (which is the default), then we don't need to do this, you can access it statically from anywhere like 
```
GameManager.doSomething()
```